lib/busser/runner_plugin/serverspec.rb

```diff
@@ -57,8 +57,7 @@ def install_serverspec
     Gem::Specification.reset
     if Array(Gem::Specification.find_all_by_name('serverspec')).size == 0
       if Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.0')
-        banner('Installing net-ssh < 2.10')
-        install_gem('net-ssh', '< 2.10')
+        raise "Ruby < 2.0 is EOLed and no longer supported"
```

Comment:
should be `2.2` since that's what https://github.com/test-kitchen/busser-serverspec/pull/51 was trying to fix ?